omod/src/main/webapp/resources/htmlforms/sierra_leone/section-plan.xml

```diff
@@ -179,288 +179,63 @@
-
-                <repeat>
-                    <template>
-                        <p>
-                            <obs conceptId="PIH:Lab test ordered coded"
-                                 answerConceptId="{labName}"
-                                 answerCode="{codeLabel}"
-                                 style="checkbox"/>
-                        </p>
-                    </template>
-
-                    <render labName="CIEL:163722" codeLabel="pihcore.lab.hivRapid" comment="AFB / Smear"/>
-                    <render labName="PIH:1041" codeLabel="pihcore.lab.elisaQuant" comment="HIV ELISA (quant)"/>
-                    <!-- ToDo: ELISA EID -->
-                    <render labName="PIH:HIV VIRAL LOAD" codeLabel="pihcore.lab.viralLoad" comment="HIV viral load"/>
-                    <render labName="PIH:CD4 COUNT" codeLabel="pihcore.lab.cd4" comment="CD4 count"/>
-                </repeat>
-
-                <!-- Other -->
-                <label>
-                    <uimessage code="pihcore.other"/>
-                </label>
-
-                <includeIf velocityTest="$patient.gender == 'F' ">
-                    <repeat>
-                        <template>
-                            <p>
-                                <obs conceptId="PIH:Lab test ordered coded"
-                                     answerConceptId="{labName}"
-                                     answerCode="{comment}"
-                                     style="checkbox"/>
-                            </p>
-                        </template>
-                        <render labName="CIEL:885" comment="PAP test"/>
-                    </repeat>
-                </includeIf>
-
-                <repeat>
-                    <template>
-                        <p>
-                            <obs conceptId="PIH:Lab test ordered coded"
-                                 answerConceptId="{labName}"
-                                 answerCode="{comment}"
-                                 style="checkbox"/>
-                        </p>
-                    </template>
-                    <render labName="CIEL:302" comment="pihcore.lab.urine"/>
-                    <render labName="CIEL:161454" comment="Gram stain"/>
-                    <render labName="CIEL:163620" comment="H pylori"/>
-                </repeat>
-                <p>
-                    <obs conceptId="PIH:Lab test ordered coded"
-                         answerConceptId="PIH:OTHER"
-                         answerCode="zl.ifOtherSpecify"
-                         style="checkbox"
-                         showCommentField="true" commentFieldLabel=""/>
-                </p>
-            </div>
+            <p>
+                <obs conceptId="PIH:FAMILY PLANNING COMMENT" 
+                labelText="Offered and accepted Family Planning information"  
+                answers="PIH:YES,PIH:NO" 
+                answerLabels="Yes,No" 
+                style="dropdown"
```

Comment:
I would recommend avoiding dropdowns -- it requires an extra click.  I've been moving to radio buttons for all of these. @lebbiesahr